NotadogApi/Hubs/GameHub.cs

```diff
@@ -87,6 +87,20 @@ public async Task LeaveRoom()
             await Clients.User($"{id}").SendAsync("OnRoomUpdate", null);
         }
 
+        public async Task Refresh()
+        {
+            var id = _currentUserAccessor.GetCurrentId();
+            var room = await _roomStorage.GetRoomByUserId(id);
+
+            if (room == null)
+            {
+                await Clients.User($"{id}").SendAsync("OnRoomUpdate", null);
+                return;
+            }
+
+            await Clients.User($"{id}").SendAsync("OnRoomUpdate", new RoomPayload(room));
```

Comment:
You can write "await Clients.User($"{id}").SendAsync("OnRoomUpdate", room == null ? null : new RoomPayload(room))" and delete if block.
Also there are much copy-paste in this class.